deploy_carvel/deploy/pinniped-package-rbac-concierge-concierge-rbac.yml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "concierge-install-ns"
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: "pinniped-package-rbac-concierge-sa-superadmin-dangerous"
+  namespace: "concierge-install-ns"
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: "pinniped-package-rbac-concierge-role-superadmin-dangerous"
+  namespace: "concierge-install-ns"
+rules:
+- apiGroups: ["*"]
```

Comment:
This here, the `*,*,*` is something that, if deployed, should be dialed down to specific RBAC rules.